### ISSUE TITLE
Cant't install anymore 

### ISSUE THREAD

**Issue by Fabioah** [rafaelmardojai/firefox-gnome-theme#842]
**Describe the bug**
Hi, after a fresh install of vanilla OS i cant manage to make the theme works anymore. I tried with autoinstall.sh, install.sh and the manual way too. Down I will add the screenshots of the errors given while using the 2 scripts, while trying to do it the manual way doesn't give any error but doesn't work either.

I'm using Vanilla OS with Firefox 131 Flatpak with the firefox-gnome-theme master.

Also i dont't know if it can be useful but I have 2 firefox folders: ~/.mozilla and the standard flatpak one. And while in about:support it says it is using the profile in ~/.mozilla (see screenshot), however there are no files inside that directory

![Schermata del 2024-10-07 12-06-41](https://github.com/user-attachments/assets/31818b0b-4bf6-4702-bc0e-31f8c474fd67)

![Schermata del 2024-10-07 12-04-15](https://github.com/user-attachments/assets/ea6335b9-1140-48f0-afac-6da090aa1239)
![Schermata del 2024-10-07 12-04-46](https://github.com/user-attachments/assets/835b9e23-9f65-4b14-b72a-74987c64f08e)

**Comment by rafaelmardojai** [comment]
Hmm, can't say why is failing, looks like the config file is missing after copying.

How are you getting the theme files? Are all the theme files there?

You could try with the curl script also, maybe that works.

Also, when you do the manual install, can you symlink the preferences just fine or also fails? You could also go to `about:config` and manually enable the required prefs if needed: https://github.com/rafaelmardojai/firefox-gnome-theme?tab=readme-ov-file#required-firefox-preferences

**Comment by rafaelmardojai** [comment]
> Also i dont't know if it can be useful but I have 2 firefox folders: ~/.mozilla and the standard flatpak one. And while in about:support it says it is using the profile in ~/.mozilla (see screenshot), however there are no files inside that directory

Yeah, that's normal. Flatpaked firefox is not aware of its real `.mozilla` folder path.

**Comment by Fabioah** [comment]
Okay so  I had also tried the curl script and it failed without giving errors. Now I entered the host-shell tried again with the curl script and it worked, however the normal "install" and "auto-install" scripts are still failing with the same errors. Anyway problem solved, thank you! 
![Schermata del 2024-10-07 17-08-42](https://github.com/user-attachments/assets/4e91b56b-2659-455a-86aa-9b4440439a0d)
